helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capi-aggregated-error-logs.json

```diff
@@ -52,7 +52,7 @@
             "uid": "${loki_datasource}"
           },
           "editorMode": "builder",
-          "expr": "{app=~\"cluster-api\"} |~ `(?i)error` |= `$cluster` | logfmt",
+          "expr": "{job=~\"cluster-api\"} |~ `(?i)error` |= `$cluster` | logfmt",
```

Comment:
For `app=cluster-api`, job is equal to `api-exporter`, not `cluster-api`.
You can rely on `name=cluster-api`